docs/sources/reference/cli/environment-variables.md

```diff
@@ -14,13 +14,28 @@ The following environment variables are supported:
 
 * `GODEBUG`
 * `HTTP_PROXY`
+* `HTTPS_PROXY`
+* `NO_PROXY`
 * `PPROF_MUTEX_PROFILING_PERCENT`
 * `PPROF_BLOCK_PROFILING_RATE`
 * `GOMEMLIMIT`
-
+* `AUTOMEMLIMIT`
+* `GOGC`
+* `GOMAXPROCS`
+* `GOTRACEBACK`
 
 Refer to the [Go runtime][runtime] documentation for more information about Go runtime environment variables.
 
+# Windows Service Configuration
+
+To configure any of the available environment variables when running {{< param "PRODUCT_NAME" >}} as a Windows Service, open `regedit`, the Windows Registry Editor, and navigate to the registry key `HKEY_LOCAL_MACHINE\SOFTWARE\GrafanaLabs\Alloy`. Under this key will be an `Environment` multi-string value where any desired environment variables can be provided on separate lines.
```

Comment:
It might be better that we refer to the configure topic for these steps instead of duplicating them here.

Something like this:

```
Refer to [Change environment variable values][change] for more information about changing the environment variables in Windows.

[change]: https://grafana.com/docs/alloy/latest/reference/cli/environment-variables/#change_environment_variable_values
```

If we add Linux/MacOS config instructions (like the Windows instructions) then we'd probably need to rework this a bit here